MapboxNavigation/RouteMapViewController.swift

```diff
@@ -576,6 +582,10 @@ extension RouteMapViewController: NavigationComponent {
     func navigationService(_ service: NavigationService, didRefresh routeProgress: RouteProgress) {
         mapView.show([routeProgress.route], legIndex: routeProgress.legIndex)
         if routeLineTracksTraversal {
+            mapView.updateUpcomingRoutePointIndex(routeProgress: routeProgress)
+            if let location = router.location {
+                mapView.updateTraveledRouteLine(location.coordinate)
```

Comment:
You can update `updateTraveledRouteLine(_:)` to accept optional and move `if let location = router.location` inside of it to be checked in `guard`.

Reply:
Thanks for the suggestion! I'll work on it.